content/docs/reference/config/installation-options-changes/index.md

```diff
@@ -17,443 +17,217 @@ The tables below show changes made to the installation options used to customize
-| `global.configValidation` | `true` | `true` |  | `Whether to perform server-side validation of configuration.` |
-
-### Modified `gateways` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `gateways.istio-ingressgateway.type` | `LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be` | `LoadBalancer` |  | `change to NodePort, ClusterIP or LoadBalancer if need be` |
-| `gateways.istio-egressgateway.enabled` | `true` | `false` |  |  |
-| `gateways.istio-egressgateway.type` | `ClusterIP #change to NodePort or LoadBalancer if need be` | `ClusterIP` |  | `change to NodePort or LoadBalancer if need be` |
-
-### Modified `certmanager` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `certmanager.tag` | `v0.3.1` | `v0.6.2` |  |  |
+| `global.tag` | `release-1.1-latest-daily` | `1.2.0-rc.2` | `Default tag for Istio images.`  | `Default tag for Istio images.` |
+| `global.proxy.resources.limits.memory` | `128Mi` | `1024Mi` |  |  |
+| `global.proxy.dnsRefreshRate` | `5s` | `300s` | `Configure the DNS refresh rate for Envoy cluster of type STRICT_DNS 5 seconds is the default refresh rate used by Envoy`  | `Configure the DNS refresh rate for Envoy cluster of type STRICT_DNS This must be given it terms of seconds. For example, 300s is valid but 5m is invalid.` |
 
-### Modified `kiali` key/value pairs
+### Modified `mixer` key/value pairs
 
 | Key | Old Default Value | New Default Value | Old Description | New Description |
 | --- | --- | --- | --- | --- |
-| `kiali.tag` | `istio-release-1.0` | `v0.14` |  |  |
+| `mixer.adapters.useAdapterCRDs` | `true` | `false` | `Setting this to false sets the useAdapterCRDs mixer startup argument to false`  | `Setting this to false sets the useAdapterCRDs mixer startup argument to false` |
 
-### Modified `security` key/value pairs
+### Modified `grafana` key/value pairs
 
 | Key | Old Default Value | New Default Value | Old Description | New Description |
 | --- | --- | --- | --- | --- |
-| `security.selfSigned` | `true # indicate if self-signed CA is used.` | `true` |  | `indicate if self-signed CA is used.` |
-
-### Modified `pilot` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `pilot.autoscaleMax` | `1` | `5` |  |  |
-| `pilot.traceSampling` | `100.0` | `1.0` |  |  |
+| `grafana.image.tag` | `5.4.0` | `6.1.6` |  |  |
 
 ## New configuration options
 
-### New `istio_cni` key/value pairs
-
-| Key | Default Value | Description |
-| --- | --- | --- |
-| `istio_cni.enabled` | `false` |  |
-
-### New `servicegraph` key/value pairs
+### New `tracing` key/value pairs
 
 | Key | Default Value | Description |
 | --- | --- | --- |
-| `servicegraph.nodeSelector` | `{}` |  |
+| `tracing.podAntiAffinityLabelSelector` | `[]` |  |
```

Comment:
@johnma14 @linsun Please include descriptions.

Reply:
@rcaballeromx it will take weeks to add these description fields.  These files are autogenerated from this files:

https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml

as well as all of the `values.yaml` in the subcharts.

This work would take weeks, and i don't think we want to hold the release for the lack of description fields, as we would have to spin another RC - and potentially delay the release.

Reply:
I think this is okay - we did add a link to k8s in release note for this.  This is well known for folks know k8s well.